### PR TITLE
Filipino Translation - 1 168 translated words

### PR DESCRIPTION
**Pull request by mind-blast**
Translating total 1,168 words.
undoing.asc - translated 236 words
viewing-history.asc - 932 translated words

Code text and computer code generated text are not included as translated words for this contribution.

### CODE REVIEW

File: book/02-git-basics/sections/viewing-history.asc [hunk 1]
```diff
@@ -127,14 +127,14 @@ Date:   Sat Mar 15 10:31:28 2008 -0700
...
+Naglalagay din ito ng buod ng impormasyon sa dulo.
 
-Another really useful option is `--pretty`.
-This option changes the log output to formats other than the default.
-A few prebuilt options are available for you to use.
-The `oneline` option prints each commit on a single line, which is useful if you're looking at a lot of commits.
-In addition, the `short`, `full`, and `fuller` options show the output in roughly the same format but with less or more information, respectively:
+Ang isa pang talagang kapaki-pakinabang na opsyon ay ang `--pretty`.
+Ang opsyon na ito ay nagbabago ng log output para i-format maliban sa default.
+Ang ilang prebuilt na mga opsyon ay magagamit para sa iyo upang gamitin.
```
Comment (severinolorillajr) [inline book/02-git-basics/sections/viewing-history.asc]: @robin-ho , can you recheck the source please and update accordingly?
The sentence does not make sense. :)

**Review by severinolorillajr — CHANGES_REQUESTED** [review]

File: book/02-git-basics/sections/viewing-history.asc [hunk 2]
```diff
@@ -1,18 +1,18 @@
...
+Ang mga halimbawang ito ay gumagamit ng pinaka simpleng proyekto na tinatawag na ``simplegit''.
+Upang makuha ang proyekto, patakbuhin ang
 
 [source,console]
 ----
 $ git clone https://github.com/schacon/simplegit-progit
 ----
 
-When you run `git log` in this project, you should get output that looks something like this:(((git commands, log)))
+Kapag ikaw ay nagpatakbo ng `git log` sa proyektong ito, dapat kang makakuha ng output na mukhang tulad nito:(((git commands, log)))
```
Comment (severinolorillajr) [inline book/02-git-basics/sections/viewing-history.asc]: output na parang kagaya nito.

File: book/02-git-basics/sections/viewing-history.asc [hunk 3]
```diff
@@ -36,14 +36,14 @@ Date:   Sat Mar 15 10:31:28 2008 -0700
...
+Tulad ng iyong nakikita, ang utos na ito ay naglilista sa bawat commit na kasama ang SHA-1 checksum nito, ang pangalan ng may-akda at email, ang petsa ng pagkasulat, at ang mensahe ng commit.
 
-A huge number and variety of options to the `git log` command are available to show you exactly what you're looking for.
-Here, we'll show you some of the most popular.
+Isang malaking bilang at iba't ibang mga opsyon sa `git log` na utos ay magagamit para ipakita sayo ang eksakto na kung ano ang iyong hinahanap.
+Dito, ipapakita namin sa iyo ang ilang mga pinaka sikat.
 
-One of the more helpful options is `-p` or `--patch`, which shows the difference (the _patch_ output) introduced in each commit.
-You can also limit the number of log entries displayed, such as using `-2` to show only the last two entries.
+Isa sa maraming matulungin na mga opsyon ay ang `-p` o `--patch`, na nagpapakita ng pagkakaiba (ang _tapal_ na output) ay ipinakilala sa bawat commit.
```
Comment (severinolorillajr) [inline book/02-git-basics/sections/viewing-history.asc]: matulungin to nakakatulong

File: book/02-git-basics/sections/viewing-history.asc [hunk 4]
```diff
@@ -127,14 +127,14 @@ Date:   Sat Mar 15 10:31:28 2008 -0700
  3 files changed, 54 insertions(+)
 ----
 
-As you can see, the `--stat` option prints below each commit entry a list of modified files, how many files were changed, and how many lines in those files were added and removed.
-It also puts a summary of the information at the end.
+Tulad ng iyong nakikita, ang `--stat` ang opsyon ay nagpapakita sa ibabaw sa bawat commit na entry sa listahan ng mga binagong mga file, kung gaano karaming mga file ang binago, at gaano karami ang mga linya sa mga file ang idinagdag at inalis.
```
Comment (severinolorillajr) [inline book/02-git-basics/sections/viewing-history.asc]: ang `--stat` na opsyon

File: book/02-git-basics/sections/viewing-history.asc [hunk 5]
```diff
@@ -127,14 +127,14 @@ Date:   Sat Mar 15 10:31:28 2008 -0700
...
+Naglalagay din ito ng buod ng impormasyon sa dulo.
 
-Another really useful option is `--pretty`.
-This option changes the log output to formats other than the default.
-A few prebuilt options are available for you to use.
-The `oneline` option prints each commit on a single line, which is useful if you're looking at a lot of commits.
-In addition, the `short`, `full`, and `fuller` options show the output in roughly the same format but with less or more information, respectively:
+Ang isa pang talagang kapaki-pakinabang na opsyon ay ang `--pretty`.
+Ang opsyon na ito ay nagbabago ng log output para i-format maliban sa default.
```
Comment (severinolorillajr) [inline book/02-git-basics/sections/viewing-history.asc]: default to nakatakda.

File: book/02-git-basics/sections/viewing-history.asc (hunk 1 above)
Comment (mind-blast) [reply book/02-git-basics/sections/viewing-history.asc]: @severinolorillajr will check on it.

File: book/02-git-basics/sections/viewing-history.asc [hunk 6]
```diff
@@ -202,34 +202,34 @@ $ git log --pretty=format:"%h %s" --graph
 *  11d191e Merge branch 'defunkt' into local
 ----
 
-This type of output will become more interesting as we go through branching and merging in the next chapter.
+Ang uri ng output ay magiging mas kawili-wili habang tayo ay gumagamit ng pag-branching at pag-merging sa susunod na kabanata.
```
Comment (severinolorillajr) [inline book/02-git-basics/sections/viewing-history.asc]: Ang tipo ng output na ito ay...

File: book/02-git-basics/sections/viewing-history.asc [hunk 7]
```diff
@@ -202,34 +202,34 @@ $ git log --pretty=format:"%h %s" --graph
...
 
-In addition to output-formatting options, `git log` takes a number of useful limiting options -- that is, options that let you show only a subset of commits.
-You've seen one such option already -- the `-2` option, which displays only the last two commits.
-In fact, you can do `-<n>`, where `n` is any integer to show the last `n` commits.
-In reality, you're unlikely to use that often, because Git by default pipes all output through a pager so you see only one page of log output at a time.
+Sa karagdagan sa output-formatting na mga opsyon, ang `git log` ay kumukuha ng isang bilang ng kapaki-pakinabang na paglilimit na mga opsyon -- yan ay, ang mga opsyon na nagpapakita sa iyo lamang ng subset ng mga commit.
+Nakita mo na ang opsyon na iyon -- ang `-2` na opsyon, na nagpapakita lamang sa huling dalawang commit.
+Sa katunayan, maaari mong gawin `-<n>`, kung saan `n` ang anumang integer na ipinakita sa huling `n` na mga commit.
+Sa totoo, ikaw ay malamang na hindi gumagamit nang madalas, dahil ang Git sa pamamagitan ng default na mga pipe sa lahat ng output sa pamamagitan ng isang pager kaya nakikita lamang ang isang pahina sa log output sa isang pagkakataon.
```
Comment (severinolorillajr) [inline book/02-git-basics/sections/viewing-history.asc]: malamang hindi ka gumagamit niyan ng madalas

File: book/02-git-basics/sections/undoing.asc [hunk 8]
```diff
@@ -125,17 +125,17 @@ Changes to be committed:
...
+Nakikita mo na ang mga pagbabago ay naibalik na.
 
-[IMPORTANT]
+[MAHALAGA]
 =====
-It's important to understand that `git checkout -- <file>` is a dangerous command.
-Any changes you made to that file are gone -- Git just copied another file over it.
-Don't ever use this command unless you absolutely know that you don't want the file.
+Ito ay mahalagang maunawaan na ang `git checkout -- <file>` ay isang delikadong utos.
+Anumang mga pagbabago na iyong ginawa sa file ay nawala -- Ang Git ay naka-kopya ng ibang file sa ganito.
```
Comment (severinolorillajr) [inline book/02-git-basics/sections/undoing.asc]: ay mawawala.. future tense.

File: book/02-git-basics/sections/undoing.asc [hunk 9]
```diff
@@ -125,17 +125,17 @@ Changes to be committed:
...
+Huwag kailanman gumamit sa utos maliban kung ikaw ay talagang alam na hindi mo gusto ang file.
 =====
 
-If you would like to keep the changes you've made to that file but still need to get it out of the way for now, we'll go over stashing and branching in <<_git_branching#_git_branching>>; these are generally better ways to go.
+Kung nais mong panatilihin ang pagbabago ay nagawa mo na ang file na iyon ngunit kailangan mo pa rin na lumabas daanan sa ngayon, pupunta tayo sa stashing at branching sa <<_git_branching#_git_branching>>; ang mga ito ay pangkalahatang mas mahusay na paraan upang pumunta.
```
Comment (severinolorillajr) [inline book/02-git-basics/sections/undoing.asc]: Kung nais mong panatilihin ang pagbabago na nagawa mo sa file na iyon

File: book/02-git-basics/sections/undoing.asc [hunk 10]
```diff
@@ -125,17 +125,17 @@ Changes to be committed:
...
 
-Remember, anything that is _committed_ in Git can almost always be recovered.
-Even commits that were on branches that were deleted or commits that were overwritten with an `--amend` commit can be recovered (see <<_git_internals#_data_recovery>> for data recovery).
-However, anything you lose that was never committed is likely never to be seen again.
+Tandaan, kahit ano ang na _committed_ sa Git ay maaaring halos palaging mababawi.
```
Comment (severinolorillajr) [inline book/02-git-basics/sections/undoing.asc]: kahit ano ang _na-commit_ sa Git

File: book/02-git-basics/sections/undoing.asc [hunk 11]
```diff
@@ -125,17 +125,17 @@ Changes to be committed:
...
+Kahit na ang mga commit na nasa branch na natanggal na o mga commit na napalitan na nang `--amend` ang commit ay pwedeng mababawi (tingnan <<_git_internals#_data_recovery>> para sa pagbawi ng datos).
```
Comment (severinolorillajr) [inline book/02-git-basics/sections/undoing.asc]: ay napalitan na ng

**Comment by severinolorillajr** [comment]
Hi @robin-ho , please check the review.
And let me know once you have already updated them so we can merge this.
Thanks.

File: book/02-git-basics/sections/viewing-history.asc (hunk 2 above)
Comment (mind-blast) [reply book/02-git-basics/sections/viewing-history.asc]: done change.

File: book/02-git-basics/sections/viewing-history.asc (hunk 3 above)
Comment (mind-blast) [reply book/02-git-basics/sections/viewing-history.asc]: done change

File: book/02-git-basics/sections/viewing-history.asc (hunk 4 above)
Comment (mind-blast) [reply book/02-git-basics/sections/viewing-history.asc]: done change

File: book/02-git-basics/sections/viewing-history.asc (hunk 5 above)
Comment (mind-blast) [reply book/02-git-basics/sections/viewing-history.asc]: done change

File: book/02-git-basics/sections/viewing-history.asc (hunk 6 above)
Comment (mind-blast) [reply book/02-git-basics/sections/viewing-history.asc]: done change

File: book/02-git-basics/sections/viewing-history.asc (hunk 7 above)
Comment (mind-blast) [reply book/02-git-basics/sections/viewing-history.asc]: done change

File: book/02-git-basics/sections/viewing-history.asc (hunk 1 above)
Comment (mind-blast) [reply book/02-git-basics/sections/viewing-history.asc]: Original sentence : A few prebuilt options are available for you to use.
Change translation to : Ang kaunting prebuilt na mga opsyon ay magagamit para gamitin mo.

File: book/02-git-basics/sections/undoing.asc (hunk 8 above)
Comment (mind-blast) [reply book/02-git-basics/sections/undoing.asc]: done change

File: book/02-git-basics/sections/undoing.asc (hunk 9 above)
Comment (mind-blast) [reply book/02-git-basics/sections/undoing.asc]: done change

File: book/02-git-basics/sections/undoing.asc (hunk 10 above)
Comment (mind-blast) [reply book/02-git-basics/sections/undoing.asc]: done change

File: book/02-git-basics/sections/undoing.asc (hunk 11 above)
Comment (mind-blast) [reply book/02-git-basics/sections/undoing.asc]: done change

**Comment by mind-blast** [comment]
@severinolorillajr Already done revisions for this contribution.  Thank you.

**Pull request merged**